src/discord/commands/moderation/nickname.ts

```diff
@@ -0,0 +1,41 @@
+import { Command, AccessLevel, CommandError, Logger, Guards } from "dd-botkit";
+
+export const NicknameCommand: Command = {
+    opts: {
+        name: "nickname",
+        access: AccessLevel.MODERATOR,
+        category: "Moderation",
+        guards: [
+            Guards.Argumented(
+                "nickname",
+                "Force changes the nickname of a user",
+                [
+                    {
+                        name: "user",
+                        type: "user",
+                        required: true
+                    },
+                    {
+                        name: "nickname",
+                        type: "string",
+                        required: true
+                    }
+                ]
+            )
+        ]
+    },
+    handler: async (msg, next) => {
+        const [user, nickname] = msg.args;
+        const member = msg.guild.members.get((msg.args[0] as User).id);
```

Comment:
You already destructured the array, you have the user variable declared. Use that.